litmus-portal/graphql-server/pkg/cluster/cluster_jwt.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/litmuschaos/litmus/litmus-portal/graphql-server/utils"
 )
 
-// ClusterCreateJWT generates jwt used in cluster registration
-func ClusterCreateJWT(id string) (string, error) {
+// CreateJWT generates jwt used in cluster registration
+func CreateJWT(id string) (string, error) {
```

Comment:
@namkyu1999  can we keep the function name as it is `ClusterCreateJWT` or `CreateClusterJWT` since is scoped to the cluster create operation. `CreateJWT` is very much related to authorization-related operations which might create some confusion.